android/sdk/src/main/java/com/taobao/weex/ui/module/WXTimerModule.java

```diff
@@ -34,28 +39,20 @@
 import com.taobao.weex.dom.action.Actions;
 import com.taobao.weex.utils.WXJsonUtils;
 import com.taobao.weex.utils.WXLogUtils;
-
 import java.util.ArrayList;
 import java.util.HashMap;
 
-import static com.taobao.weex.bridge.WXBridgeManager.KEY_ARGS;
-import static com.taobao.weex.bridge.WXBridgeManager.KEY_METHOD;
-import static com.taobao.weex.bridge.WXBridgeManager.METHOD_CALLBACK;
-import static com.taobao.weex.bridge.WXBridgeManager.METHOD_CALL_JS;
-
 public class WXTimerModule extends WXModule implements Destroyable, Handler.Callback {
 
   private final static String TAG = "timer";
   private Handler handler;
+  private SparseArray<Integer> antiIntAutoBoxing;
```

Comment:
antiIntAutoBoxing is enough. 

* JIRA is down, BTW.
* JIRA waring is irrelevant.